src/app/routines/myRoutines/MyRoutines.scss

```diff
@@ -0,0 +1,28 @@
+@import '@/styles/base/variables';
+
+.routine-list-wrapper {
+  display: grid;
```

Comment:
it would be better to use flex with wrap option instead of fixed grid. It kinda ruins a "flexible" approach of app